client/js/app.js

```diff
@@ -1,7 +1,11 @@
 const TableModel = require ('./table-model');
 const TableView = require('./table-view');
+const ButtonView = require('./button-view');
```

Comment:
Use variable names that would intuitively describe the story to a reader of your code. I understand what to expect from TableView, but not from "ButtonView" - is it just a screen with a button?